docs/source/algorithms/SumOverlappingTubes-v1.rst

```diff
@@ -0,0 +1,86 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm takes an input workspace, a group workspace or a selection of workspaces, with a PSD tube instrument, and converts the workspace to a one or two-dimensional workspace of counts as a function of height and scattering angle. There are options to either see the data as it is, or apply a correction to straighten the Debye-Scherrer cones.
+
+This algorithm is written for the geometry of D2B, where the detector tubes have the same radial distance from the sample, and rotate around the sample in the horizontal plane during a detector scan. This algorithm could be generalised to support other instrument geometries in the future.
+
+General Behavior
+#################
+
+For the output workspace a grid is made, the x-axis is the scattering angle and the y-axis is the tube height. Counts from the input workspaces are then put onto this grid. Where one of the entries in the scattering angle is between two angles the count are split between the two, unless the ScatteringAngleTolerance option is set.
```

Comment:
The values on the y-axis can be negative. Is `tube height` the right name for them?

Reply:
Hmm, a good question. I think using height is consistent with the terminology used at the ILL, so I would prefer to leave it as that for now. I will clarifiy what this means here a bit though.